Tests/PlayMode/ZoomDhvCameraTest.cs

```diff
@@ -0,0 +1,136 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using ReupVirtualTwinTests.utils;
+using Unity.Cinemachine;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.TestTools;
+
+namespace ReupVirtualTwin.behaviours
+{
+    public class ZoomDhvCameraTest
+    {
+        ReupSceneInstantiator.SceneObjects sceneObjects;
+        InputTestFixture input;
+        Mouse mouse;
+        Touchscreen touch;
+        ZoomDhvCamera zoomDhvCamera;
+        CinemachineCamera dhvCamera;
```

Comment:
it took me a while to understand what is the purpose for these 2 variables. More descriptive names would have been appreciated.
```suggestion
        ZoomDhvCamera zoomDhvBehavior;
        CinemachineCamera dhvCineMachineComponent;
```